src/controllers/login.controller.ts

```diff
@@ -7,6 +8,8 @@ import { JwtResult } from '../typings/jwt.typings';
 export class LoginController {
   constructor(private loginService: LoginService) {}
 
+  @ApiTags('Login')
+  @ApiBody({ type: LoginDTO })
   @Post()
   @HttpCode(HttpStatus.OK)
   login(@Body() loginDTO: LoginDTO): Promise<JwtResult> {
```

Comment:
For this method can you add description about what is this method and common responses like 200, 400 and 401?

Reply:
Resolved on f7fa703f15b2a2b058a419175eccb0b8faca3f8e.